tools/build-image.sh

```diff
@@ -85,7 +85,9 @@ if ! kiwi-ng --version &>/dev/null ; then
   error_exit "missing kiwi-ng"
 fi
 
-if ! /sbin/mkfs.btrfs --version &>/dev/null ; then
+
+MKFS_BTRFS=`which mkfs.btrfs`
```

Comment:
Apparently, using `which` introduces a new dependency, and I've been advised to refrain from using it.